moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -37,67 +30,62 @@ export const PROJECT_LIST_VIEW_QUERY_CONFIG = {
     defaultColumn: "updated_at",
     defaultDirection: "desc",
   },
-  filters: PROJECT_LIST_VIEW_FILTERS_CONFIG,
-  export: PROJECT_LIST_VIEW_EXPORT_CONFIG,
   search: {
     placeholder:
       "Search by ID, name, description, phase, lead, sponsor, partners, eCAPRIS ID...",
     defaultFieldsOperator: "_or",
   },
   columns: {
```

Comment:
Lots of changes in this config so I'll try to cover them here
- GridTable config no longer needed: `hidden`, `primary_key`, `icon`, `width`, `filter` (set in the Material Table config so this was sort of a duplicate)